clang/include/clang/Serialization/ASTBitCodes.h

```diff
@@ -1502,6 +1509,12 @@ enum DeclCode {
   /// An ImplicitConceptSpecializationDecl record.
   DECL_IMPLICIT_CONCEPT_SPECIALIZATION,
 
+  // A decls specilization record.
```

Comment:
Note `specialization` is misspelled in this comment and the one below.

Reply:
Done